### ISSUE TITLE
Publish sources for opentelemetry-aws-resources

### ISSUE THREAD

**Issue by danielcompton** [open-telemetry/opentelemetry-java-contrib#750]
**Is your feature request related to a problem? Please describe.**

I use IntelliJ. When I try to look at the implementation of classes in opentelemetry-aws-resources, IntelliJ offers to download sources. However, it then returns an error because sources are not available.

**Describe the solution you'd like**

It looks like this should be working already, based on https://github.com/open-telemetry/opentelemetry-java-contrib/commit/4bfd97dfff1ad7d58898a9e0aa2a43ce82794f3a#diff-c1b454e4fc67605fd57a091ec64b9333081ced21d02b36e839ec18c67a03e265R18

and https://repo1.maven.org/maven2/io/opentelemetry/contrib/opentelemetry-aws-resources/1.22.0-alpha/ showing the source JARs are available.

**Describe alternatives you've considered**

**Additional context**

I'm using io.opentelemetry.contrib/opentelemetry-aws-resources version "1.22.0-alpha"

<img width="401" alt="Screenshot of IntelliJ IDEA (21-02-23, 12-21-51 PM)" src="https://user-images.githubusercontent.com/811954/220212109-9d83a02d-abf8-4fc3-acf8-334eb49f4e0a.png">

**Comment by trask** [comment]
hi @danielcompton!

I'm seeing the sources jar in the usual place for this artifact:

http://repo1.maven.org/maven2/io/opentelemetry/contrib/opentelemetry-aws-resources/1.22.0-alpha/opentelemetry-aws-resources-1.22.0-alpha-sources.jar

can you check where Intellij is trying to download the source from?

**Comment by danielcompton** [comment]
@trask sorry about that, as I did some more digging I also saw that the sources are available. 

It seems like IntelliJ has the same issue for any library with a `-alpha` version.

I was able to download the sources with

```
 mvn dependency:sources dependency:resolve -Dclassifier=sources

...
[INFO]    io.opentelemetry.contrib:opentelemetry-aws-resources:jar:sources:1.22.0-alpha:compile
```

So this looks like an IntelliJ issue, not one of how these are published. Sorry for the noise!

I'm writing in Clojure, using Cursive, so it's possible that this might be a contributing factor too.
